### PR TITLE
Add Premade Merc Roles for admenes

### PR DESCRIPTION
**Pull request by Tryded**
## About the PR

adds cool loadouts for admenes to use

There are non-EVA and EVA variants
## Why / Balance

there are 4 main roles:
Squad Leader: Armed with L6 Saw, pinpointer, and flares. Their job is to lead their squad while occasionally giving suppressive fire.

Sergeant: Assists Squad Leader with leadership and is armed with an AKM and plastic explosives for breaking into rooms.

Mercenary Corpsman: Does medic stuff

Mercenary Rifleman: Armed with an AKM and has the main task of following the Leader's orders. They shoot stuff.

## Technical details

YAML only and doesn't touch upstream code
## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

**Comment by Tryded** [comment]
Will finish FTL files and EVA stuff tommorrow

**Comment by Tryded** [comment]
what does these errors mean :elp:

**Comment by Tryded** [comment]
ready for review

File: Resources/Locale/en-US/deltav/job/job-description.ftl
Comment (Colin-Tel) [inline Resources/Locale/en-US/deltav/job/job-description.ftl:1]: This file shouldn't be touched.

File: Resources/Locale/en-US/deltav/job/job-names.ftl [hunk 1]
```diff
@@ -1,2 +1,3 @@
 job-name-medical-borg = Medical Cyborg
 job-name-courier = Courier
+job-name-mercsquadleader = Mercenary Squad Leader
```
Comment (Colin-Tel) [inline Resources/Locale/en-US/deltav/job/job-names.ftl]:
```suggestion
job-name-mercsquadleader = Mercenary Squad Leader

```

File: Resources/Locale/en-US/deltav/job/job-supervisors.ftl [hunk 2]
```diff
@@ -0,0 +1,3 @@
+job-supervisors-contractors = the contractors
+job-supervisors-mercsquadleader = the squad leader
+job-supervisors-mercsergeant = the sergeant
```
Comment (Colin-Tel) [inline Resources/Locale/en-US/deltav/job/job-supervisors.ftl]:
```suggestion
job-supervisors-mercsergeant = the sergeant

```

File: Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/backpack.yml [hunk 3]
```diff
@@ -32,3 +32,72 @@
...
+  - type: StorageFill
+    contents:
+    - id: BoxSurvivalEngineering
+    - id: MagazineLightRifle
+    - id: MagazineLightRifle
+    - id: MagazineLightRifle
+    - id: MagazineLightRifle
+    - id: CrowbarRed
+    - id: Zipties
+    - id: Zipties
```
Comment (Colin-Tel) [inline Resources/Prototypes/DeltaV/Catalog/Fills/Backpacks/StarterGear/backpack.yml]:
```suggestion
    - id: Zipties
    
```

File: Resources/Prototypes/DeltaV/Entities/Objects/Misc/identification_cards.yml [hunk 4]
```diff
@@ -10,3 +10,31 @@
...
+  id: MercSquadLeaderIDCard
+  name: squad leader ID card
+  components:
+  - type: Sprite
+    layers:
+    - state: default
+  - type: Access
+    tags:
+    - Maintenance
+    - External
```
Comment (Colin-Tel) [inline Resources/Prototypes/DeltaV/Entities/Objects/Misc/identification_cards.yml]:
```suggestion
    - External
    
```

File: Resources/Prototypes/DeltaV/Roles/Jobs/Fun/merc_startinggear.yml [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+    shoes: ClothingShoesBootsMerc
+    head: ClothingHeadHelmetMerc
+    eyes: ClothingEyesGlassesMercenary
+    gloves: ClothingHandsMercGlovesCombat
+    outerClothing: ClothingOuterHardsuitMerc
+    suitstorage: WeaponRifleAk
+    id: MercRiflemanIDCard
+    ears: ClothingHeadsetFreelance
+    belt: ClothingBeltMercWebbing
+    pocket1: FlashlightSeclite
```
Comment (Colin-Tel) [inline Resources/Prototypes/DeltaV/Roles/Jobs/Fun/merc_startinggear.yml]:
```suggestion
    pocket1: FlashlightSeclite
    
```

File: Resources/Prototypes/_NF/Entities/Clothing/Head/hardsuit-helmets.yml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  - type: PressureProtection
+    highPressureMultiplier: 0.525
+    lowPressureMultiplier: 10000
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.9
+        Slash: 0.9
+        Piercing: 0.9
+        Heat: 0.9
```
Comment (Colin-Tel) [inline Resources/Prototypes/_NF/Entities/Clothing/Head/hardsuit-helmets.yml]:
```suggestion
        Heat: 0.9
        
```

File: Resources/Prototypes/_NF/Entities/Clothing/OuterClothing/hardsuits.yml [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+        Slash: 0.9
+        Piercing: 0.5
+        Heat: 0.9
+        Radiation: 0.5
+        Caustic: 0.8
+  - type: ClothingSpeedModifier
+    walkModifier: 0.85
+    sprintModifier: 0.9
+  - type: ToggleableClothing
+    clothingPrototype: ClothingHeadHelmetHardsuitMerc
```
Comment (Colin-Tel) [inline Resources/Prototypes/_NF/Entities/Clothing/OuterClothing/hardsuits.yml]:
```suggestion
    clothingPrototype: ClothingHeadHelmetHardsuitMerc
    
```

**Review by Colin-Tel — CHANGES_REQUESTED** [review]
Some minor .yml housekeeping

**Comment by Lyndomen** [comment]
Love the idea here, headmin approved.

File: Resources/Locale/en-US/deltav/job/job-description.ftl
Comment (Tryded) [reply Resources/Locale/en-US/deltav/job/job-description.ftl:1]: I thought I reverted all of these.. hold on

**Comment by Tryded** [comment]
No idea why tests are failing, something to do with nuke ops

**Comment by TadJohnson00** [comment]
Is this intended to replace nukeops?

**Comment by Lyndomen** [comment]
> Is this intended to replace nukeops?

Nope, just prefabbed spawns for admin events.

**Comment by TadJohnson00** [comment]
In that case, I strongly suggest avoiding the AKMS, it's unbalanced for most intents and purposes. Use the Tenebra and Pollock instead, they were both custom made for mercenaries to use as rough approximates to station weapons. Same goes for the SAW, but at least its power and capacity are limited by its accuracy. and unwieldiness.

**Comment by Tryded** [comment]
Wait why did it close

**Comment by Tryded** [comment]
All my code disappeared after I accidentally discarded all changes on the branch.. bruh

**Pull request closed without merging**